app/view_adapters/view_adapter_registry.rb

```diff
@@ -17,6 +17,7 @@ def get(type)
       "medical_safety_alert" => MedicalSafetyAlertViewAdapter,
       "raib_report" => RaibReportViewAdapter,
       "vehicle_recalls_and_faults_alert" => VehicleRecallsAndFaultsAlertViewAdapter,
-    }.fetch(type)
+       "asylum_support_decision" => AsylumSupportDecisionViewAdapter,
+   }.fetch(type)
```

Comment:
Commiting change that corrects indenting, and makes mapping a constant hash with sorted keys.